lib/auth/saml.go

```diff
@@ -111,7 +111,18 @@ func (a *Server) CreateSAMLAuthRequest(req services.SAMLAuthRequest) (*services.
 	}
 
 	req.ID = attr.Value
-	req.RedirectURL, err = provider.BuildAuthURLFromDocument("", doc)
+
+	// workaround for Ping: Ping expects `SigAlg` and `Signature` query
```

Comment:
Capitalization and punctuation needed here. Comments are full sentences.

Reply:
Done, thanks!